web/config/sync/page_manager.page.profile.yml

```diff
@@ -0,0 +1,15 @@
+uuid: c8349b6a-d910-491d-961a-2667be7aae70
+langcode: en
+status: true
+dependencies: {  }
+id: profile
+label: Profile
+use_admin_theme: true
+path: '/admin/content/dbcdk-community/profiles/{username}'
+access_logic: and
+access_conditions: {  }
```

Comment:
Waiting for pull #25 to be merged due to patches to allows permissions in conditions